tests/models/translationproject.py

```diff
@@ -116,3 +116,22 @@ def test_tp_stats_created_from_template(tutorial, fish, templates):
     assert stats['fuzzy'] == 0
     assert stats['suggestions'] == 0
     assert stats['critical'] == 0
+
+
+@pytest.mark.django_db
+def test_tp_checker(tp_checker_tests):
+    from translate.filters import checks
+    from pootle_translationproject.models import TranslationProject
+
+    from pootle_language.models import Language
```

Comment:
actually - on that note - i think its safe to do all of these imports at the top. Its only in fixtures that its not safe